src/microbit/pxt_snake/snake.md

```diff
@@ -0,0 +1,523 @@
+---
+title: "PXT: Snake"
+author: Håvard Nygård Jakobsen
+language: nb
+---
+
+![Title](Title.png)
+
+# Introduksjon {.intro}
+
+En eller annen variant av Snake har eksistert på datamaskiner helt siden slutten av 1970-tallet. 
+Mange voksne kjenner spillet fra Nokias mobiltelefoner, mens mange barn kjenner det fra moderne versjoner som slither.io.
+
+I spillet styres en slange rundt på skjermen, og slangen må unngå å krasje i kanten av skjermen og seg selv. 
+Slangen vokser når den spiser mat som dukker opp tilfeldige steder, og spillet går fortere og fortere etterhvert som slangen vokser.
+
+I denne oppgaven bruker vi engelsk som navn på blokkene og variabler. Dette er bla. for at det skal være lettere å finne 
+igjen innebygde funksjoner når vi bytter mellom blokkprogramering og javascript siden funksjonene har engelske navn i javascript. 
+Det er veldig vanlig å bruke engelske navn på funksjoner og variabeler blant programmerere. Dette gjør det bla. 
+enklere å poste kode på internettforum og få hjelp fra hele verden.
+
+Denne oppgaven er forholdsvis lang, men vi tar det skritt for skritt og forklarer underveis. La oss sette i gang.
+
+
+# Steg 1: Tegne slangen {.activity}
+
+Det første vi trenger er en liten kodesnutt som tegner slangen vår. Skjermen vår består av 5x5 ledlys. Disse kan vi skru av og på som vi vil med litt kode. 
+For å tegne slangen trenger vi noe som kan passe på hvor vi skal tegne slangen. Til dette skal vi bruke en array, en type variabel som inneholder en liste med verdier. 
+I listen vår bruker vi to verdier for å tegne en bit av slangen, den første verdien sier hvilken rekke bortover (X) vi skal tegne i og den neste sier 
+hvilken kolonne nedover (Y) vi skal tegne i. Sammen gir dette oss en X,Y posisjon til leden som vi skal skru på. 
+![Bilde av koordinater på microbit](koordinater.png)
+
+
+## Steg for steg {.check}
+
+- [ ] Først starter vi med å lage en array og sette de første verdiene. Arrays finne du under avansert. 
+Finn blokken "set list to" og endre variabelnavnet til snake. Legg den inn i "on start" blokken. 
```

Comment:
det samme kan sies om blokker. kanskje fet skrift er bedre enn hermetegn, for å gjøre det tydeligere i teksten? er litt usikker på hva som er best.